pushapkscript/script.py

```diff
@@ -62,11 +63,55 @@ def _get_product_config(context, android_product):
     except KeyError:
         raise ConfigValidationError('"products" is not part of the configuration')
 
-    try:
-        return products[android_product]
-    except KeyError:
-        raise TaskVerificationError('Android "{}" does not exist in the configuration of this instance.\
-    Are you sure you allowed to push such APK?'.format(android_product))
+    matching_products = [product for product in products if android_product in product['product_names']]
+
+    if len(matching_products) == 0:
+        raise TaskVerificationError('Android "{}" does not exist in the configuration of this '
```

Comment:
Nit: We may want to output what were the allowed product, in order to help debugging when the task fail.

[Can be done in a follow up]

Reply:
Good point, I created #91